### PR TITLE
Add a menu page; fixes 13

### PR DESCRIPTION
**Pull request by justineatrhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/COMET-WEB-Community-Edition/pulls) open
- [x] I have verified that I am following the COMET-WEB [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/COMET-WEB-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description
Add a menu page in the application and create buttons to access different applications.

### CODE REVIEW

File: COMETwebapp/COMETwebapp/Pages/Model.razor [hunk 1]
```diff
@@ -38,6 +53,20 @@ Copyright (c) 2022 RHEA System S.A.
     [Parameter]
     public string modelName { get; set; }
```
Comment (alexatrhea) [inline COMETwebapp/COMETwebapp/Pages/Model.razor:54]: Publuc properties we start with a Capital letter

**Review by alexatrhea — APPROVED** [review]
One small change needed but you can do it later if you want

File: COMETwebapp/COMETwebapp/Pages/ParameterEditor.razor [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+@page "/ParameterEditor"
+
+<ApplicationTemplate Title="Parameter Editor">
+    <Body>
+        <ParameterTable />
+    </Body>
+</ApplicationTemplate>
+
+
```
Comment (samatrhea) [inline COMETwebapp/COMETwebapp/Pages/ParameterEditor.razor:31]: these empty lines can go

**Review by samatrhea — APPROVED** [review]
an editorial change that does not require re-review

**Pull request closed without merging**